docker/web/Dockerfile

```diff
@@ -51,10 +25,9 @@ RUN set -x && apt-get update -qq \
     libgit2-dev \
     libssl1.1 \
     libgvc6 \
-    libmagic-dev \
     libthrift-dev \
     libodb-sqlite-dev \
-    ctags \
+    libodb-pgsql-dev \
```

Comment:
`libodb-pgsql-dev` shall be a dependency, otherwise the image would be unusable with PgSQL.